hazelcast/src/test/java/com/hazelcast/test/annotation/LiveTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+
+/**
+ * Annotates tests which requires integration tests over a live api.
+ * <p/>
+ * Will be executed in nightly builds and for jclouds live tests
+ */
+public final class LiveTest {
```

Comment:
With Cloud API you mean AWS, right? We had this kind of issue quite often with Simulator nightly builds with our Provisioner tool which uses jclouds. Danny switched the test setup to use AWS command line tools which is more reliable. So I cannot really recommend to use this in an automated test.

For Simulator I went down the hard road and mocked the `ComputeService` to test the Provisioner tool without any need for AWS to be working. This even works completely offline. The trade-off here is heavy usage of mocking. So whenever there is a significant change in the jclouds API we have to adjust the tests as well.